app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,18 @@
+module Types
+  class UserType < Types::BaseObject
+    field :id, ID, null: false
+    field :image, String, null: false
+    field :username, String, null: false
+    field :first_name, String, null: false
+    field :last_name, String, null: false
+    field :street, String, null: false
+    field :city, String, null: false
+    field :state, String, null: false
+    field :zip, String, null: false
+    field :billing_info, String, null: false
```

Comment:
Will FE ever need access to billing_info? It seems like that's something the FE wouldn't really play a role in and might pose a security risk if we expose that data.

Reply:
It seems like the Stripe API handles the user billing info by returning tokens for each transaction, therefore taking away the need for us to save their billing info to our database. One thing though that I don't remember if we've talked about this already or not - there is a Stripe API that says it's organized around REST to return JSON, but there is also Stripe.js and a Stripe gem for Ruby on Rails.

Reply:
Second what Cam said re: saving to database.